Detectors/ITSMFT/common/workflow/src/STFDecoderSpec.cxx

```diff
@@ -103,7 +103,13 @@ void STFDecoder<Mapping>::run(ProcessingContext& pc)
   double timeCPU0 = mTimer.CpuTime(), timeReal0 = mTimer.RealTime();
   mTimer.Start(false);
   mDecoder->startNewTF(pc.inputs());
-  auto orig = o2::header::gDataOriginITS;
+  auto detID = Mapping::getDetID();
+  o2::header::DataOrigin orig;
+  if (detID == o2::detectors::DetID::ITS) {
+    orig = o2::header::gDataOriginITS;
+  } else {
+    orig = o2::header::gDataOriginMFT;
+  }
```

Comment:
You can use directly [``detID.getDataOrigin()``](https://github.com/AliceO2Group/AliceO2/blob/11f6a7db21bf3cad71a06a4ffbfecf3027a13ed2/DataFormats/Detectors/Common/include/DetectorsCommonDataFormats/DetID.h#L90)

Reply:
Hi Ruben,

I don't understand, detID is an integer.

Reply:
I meant ``DetID::getDataOrigin( Mapping::getDetID() )``. But I see that I've already implemented direct alias ``Mapping::getOrigin()``